pkg/api/types.go

```diff
@@ -339,7 +339,7 @@ type PersistentVolumeClaimList struct {
 	Items                []PersistentVolumeClaim `json:"items"`
 }
 
-// PersistentVolumeClaimSpec describes the common attributes of storage devices
+// Describes the common attributes of storage devices
```

Comment:
`Represents the request attributes a user can make for a claim to match a persistent volume.`